recipe-server/normandy/studies/tests/api/v2/test_api.py

```diff
@@ -51,3 +53,50 @@ def test_detail_sets_no_cookies(self, api_client):
         res = api_client.get('/api/v2/extension/{id}/'.format(id=extension.id))
         assert res.status_code == 200
         assert res.client.cookies == {}
+
+    def _upload_extension(self, api_client, path):
+        with open(path, 'rb') as f:
+            res = api_client.post('/api/v2/extension/', {
+                'name': 'test extension',
+                'xpi': f,
+            }, format='multipart')
+        return res
+
+    def test_upload_works_webext(self, api_client):
+        path = './normandy/studies/tests/data/webext-signed.xpi'
```

Comment:
An absolute file path (maybe using `settings.BASE_DIR`) might be a bit more reliable than a relative one.